pkg/repositories/config/migrations.go

```diff
@@ -1084,6 +1084,88 @@ var NoopMigrations = []*gormigrate.Migration{
 		},
 	},
 
+	{
+		ID: "pg-noop-2023-06-07-admin-tags",
```

Comment:
Drop the `pg-noop-` prefix, this was referring to the migrations added in https://github.com/flyteorg/flyteadmin/pull/542.

Also, can you move these migrations to the last position in the list of migrations? (Remember to update the date in the name).

Reply:
updated